react-app-drawer-component/dist/styles.css

```diff
@@ -0,0 +1,59 @@
+.container {
+  max-width: 900px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+}
+
+.fa-align-justify,
+li {
+  cursor: pointer;
+}
+
+.fa-align-justify {
+  margin-left: 30px;
+  font-size: 50px;
+}
+
+.fa-earlybirds {
+  margin-left: 100px;
+  margin-top: 100px;
+  font-size: 600px;
+}
```

Comment:
You don't want to directly change the default behavior of a class from a css library.  This can cause confusion with other people working on a project or even yourself. It is better to make your own classes and use those in conjunction with the libraries classes.

Reply:
I cleaned up the CSS
![revision](https://user-images.githubusercontent.com/71291742/138383177-f4418a3e-2c74-4a86-a827-b2c03c1179f9.gif)